core/domain/role_services.py

```diff
@@ -131,3 +142,71 @@ def get_all_actions(role):
         role_actions.extend(get_all_actions(parent_role))
 
     return list(set(role_actions))
+
+
+def get_updatable_roles():
+    """Returns roles to which a user's role can be updated by admin interface.
+
+    Returns:
+        dict. A dict of all roles that a user can be given by admin.
+            key -> roles in format as stored in backed.
+            value -> roles in format to show users.
+    """
+    updatable_roles = {}
+    for role in PARENT_ROLES:
+        if role not in [feconf.ROLE_GUEST]:
+            updatable_roles[role] = get_human_readable_role(role)
+    return updatable_roles
+
+
+def get_viewable_roles():
+    """Returns roles by which users can be searched in admin interface.
+
+    Returns:
+        dict. A dict of all roles by which user's can be searched.
+            key -> roles in format as stored in backend.
```

Comment:
Not really -- I would look at docstrings of other dicts in the codebase and crib from there.

In general, though, the type of the values should be specified. So in this case I would say something like:

```
    dict. Each key of this dict is a role string, and the corresponding value is a string denoting the human-readable name of that role.
```